app/helpers/chargeback_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ChargebackHelper
+  def chargeback_details_groups(chargeback_details)
+    groups = {}
+    chargeback_details.each_with_index do |r|
+      new_group = Dictionary.gettext(r[:group], :type => :rate_detail_group, :notfound => :titleize)
+      groups[new_group] = r[:group] unless groups.key?(new_group)
+    end
+    groups
+  end
+
+  def chargeback_details_metrics(chargeback_details, group)
+    metrics = {}
+    chargeback_details.each_with_index do |r|
+      if r[:metric].nil? then r[:metric] = r[:description] end
```

Comment:
Yes, some rate details don't have :metric (like Fixed rate details, see chargeback_rate.yml), so I use the :description.